### PR TITLE
NIAD-1128: Proxy find a patient

### PR DESCRIPTION
**Pull request by ElliotB-96**
*Created predicate for find patient url
*Added env variables for find patient url
*Implemented unit tests for find patient url

### CODE REVIEW

**Review by mmackiewicz1 — APPROVED** [review]

**Review by msolecka — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -78,7 +79,6 @@ cp vars.local.sh vars.sh
 
 ### Using the helper script for Docker Compose
 ```bash
-cd docker/
```
Comment (Broom94) [inline README.md:81]: Why is this removed?

Comment (z-uddin) [reply README.md:81]: I asked Elliot to make this change.
As when you follow the read me you're already in the docker directory from step before (when copying the vars shell scripts).
Just like in the GP2GP readme.

**Review by Broom94 — APPROVED** [review]
